src/helpers.js

```diff
@@ -4,12 +4,12 @@ function checkPrBranch(prBranch) {
 }
 
 function checkPrTitle(prTitle) {
-  let prTitleRegexp = /^#?\d+\s+.+$/;
+  let prTitleRegexp = /^#?\d+[:;,/&]?\s+.+$/;
```

Comment:
Honestly, I still don't like it. While `:` makes perfect sense, I'm not that convinced about other characters. Also, GitHub supports only `Fixes: #123` in PR description, so I would opt to only allow the additional colon in PR title to mimic that behavior.

Reply:
As discussed on a weekly - I'm adding only optional colon here.